CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Changed
+
+- Upgrade to Django 4.2
```

Comment:
As this change relates to the sandbox, I don't it worth to be mentioned in the CHANGELOG.